05/app.js

```diff
@@ -1,3 +1,42 @@
 console.log('DOM');
 
 const curr = document.querySelector('.js-curr');
+
+const deleteBtn = curr.cloneNode(false);
+//console.dir(deleteBtn.classList);   //?dev to z tym consol.dir nie działa, dale jest to wyświetlane jako referencja zeminna dynamicznie. Jest jakiś inny sposób? Jakaś inna medota na objecie consol.???
+deleteBtn.classList.remove('js-curr');
+deleteBtn.classList.add('js-del');
+deleteBtn.innerText = 'usuń z koszyka';
+curr.parentElement.appendChild(deleteBtn);
+
+const siblingsCurr = [...curr.parentElement.children];
+// console.log(siblingsCurr); //?dev na HTML Collection - co to jest medtoda item?
+
+for(item = 0; item < siblingsCurr.length; item++) {
+    // console.log(siblingsCurr[item].className);
+    if(siblingsCurr[item].className !== "article__btn js-curr"){   //infO No co...? Pan się nie dziwi - sam dla siebie bratem nie jest xD hahaha A jakby to zrobić jakbym chciał zapisać tylko fragment tekstu? "(...) js-curr(...)" gdzie (...) oznaczają inne - dowolne ciagi znakó lub ich brak. Jak to zapisać?
+        siblingsCurr[item].className += " siblings"; //?dev Ja dam tu kropkę to wtedy ma źle przypisane wartości dla     console.log(siblingsCurr[item].className); A to chyba wynika z tego że pracowaliśmy na tablicy która powstała w oparciu o HTML Collection - tak? Dltego trzeba dać spacje? Tak jak dajemy spacje w getElementByClassName("js-crr ja-del")
+    };                                              //?dev Czy wg tutaj na objekcie działałaby petla for? Moglibyśmy jej użyć bez tego przekształcenia - obiekty tablicopodobe - czy po prostu użylibyśmy for in i spoko ?
```

Comment:
Zapytaj mnie proszę o to na spotkaniu bo kompletnie nie wiem o co chodzi.
PS. Może o to pytałeś? -> https://www.w3schools.com/jsref/met_document_createtextnode.asp

Reply:
Hmm... chodziło mi o to że trzeba uważać przypisywaniu dodatkowej klasy za pomocą notacji    element.className
1) to fakt że musimy pobrać już przypisane klasy i wówczas do nich oddać nową - inaczej nam się nadpisze tylko nowa 
`siblingsCurr[item].className           +=           " siblings"; `
2) to trzeba pamiętać o tym żeby przed nową klasą umieścić jeszcze spacje - mogłoby też być tak:
`siblingsCurr[item].className += " "+"siblings";`
*Bo jeśli tego nie damy to w rezultacie otrzymamy: 
`class="article__descriptionsiblings"` zamiast `class="article__description siblings" `

** A mi to jeszcze pomieszało się z tym, że jeżeli chcemy wyszukać elemeny któremu będą jednocześnie przypisane dwie lub więcej klas to za pomocą querySelectora zrobimy to tak: 
`document.querySelector('.article__btn.js-del.siblings');` - podając je ciągiem po kropce 
ale już jeśli chcemy użyć getElementsByClassName to zapiszemy to tak:
`document.getElementsByClassName('article__btn js-del siblings')` - czyli klasa za klasą po spacji 

No i właśnie... :D Teraz jak to pisze i sprawdzam, to wiem co mnie zszokowało xD 
Jeżeli zrobimy `const elementTest = document.getElementsByClassName('article__btn js-del siblings);`
To wyniki dla `console.log(elementTest );`
Jest taki:
![image](https://user-images.githubusercontent.com/63197096/104147869-5a67ff00-53d0-11eb-843b-e47669633d4f.png)

Jakim cudem taki, czemu tutaj po kropce, kombo... jak sobie to już jakoś uporządkowałem - że dla getElementXYZ oraz dla  tego przypisywania kolejnej wartości dla już określonego atrybutu jest **s p a c j a**
Tak tu teraz nagle te klasy są podane "scalone" przez kropke, a jak już dołoże do tego że nurtuje mnie pytanie dlaczego tak xD no to .... sam wiesz xD Spać nie mogę :D hahaha

A oczywiście jak użyjemy 
`const elementTest = document.querySelector('.article__btn.js-del.siblings') `w rezultacie z `console.log(elementTest);` mamy
![image](https://user-images.githubusercontent.com/63197096/104148091-1fb29680-53d1-11eb-8d64-d14c870adb7c.png)

No i przez to mi to jakoś nie styka...., raz że nie wiem dlaczgo tak jest, a dwa że rozwala mi to moją "koncepcje porządkową"
Taaaa.... taaaak... pewnie - wyjątek potwierdza regułe ;P

Reply:
Jeden plus taki, że sobie to teraz trochę usystematyzowałem. Chociaż jeszcze nie mam przekonania, czy aby na pewno dobrze? xD
ale jak damy `console.log(elementTest[0])` to już będzie normalny button